compiler-plugin/src/main/java/io/ballerina/wso2/apim/catalog/ServiceCatalogCompilerPlugin.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2024, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 * Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com) All Rights Reserved.
```

Check all the places